backend/app/database.js

```diff
@@ -470,25 +470,32 @@ module.exports.changeEmail = (req, callback) => {
  *
  * @param {*} req
  * @param {*} callback
- * Changes the username of the user in the database
+ * Changes the username of the user in the database 
+ * Returns a message if the username is taken
```

Comment:
This handler also returns a message when the username has been changed successfully. I think that should be mentioned too.